.circleci/config.yml

```diff
@@ -0,0 +1,207 @@
+version: 2.1
+
+commands:
+  early_return_for_forked_pull_requests:
+    description: >-
+      If this build is from a fork, stop executing the current job and return success.
+      This is useful to avoid steps that will fail due to missing credentials.
+    steps:
+      - run:
+          name: Early return if this build is from a forked PR
+          command: |
+            if [ -n "$CIRCLE_PR_NUMBER" ]; then
+              echo "Nothing to do for forked PRs, so marking this step successful"
+              circleci step halt
+            fi
+jobs:
+  checkout_code:
+    docker:
+      - image: circleci/clojure:lein-2.8.3
+    working_directory: ~/build
+    steps:
+      - checkout
+      - save_cache:
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - ~/build
+
+  bundle_dependencies:
+    docker:
+      - image: circleci/clojure:lein-2.8.3-node
+    working_directory: ~/build
+    steps:
+      - restore_cache:
+          keys:
+            - v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+            - v1-dependencies-{{ checksum "project.clj" }}
+            - v1-node-dependencies-{{ checksum "package.json" }}
+      - run: lein deps
+      - run: npm install
+      - save_cache:
+          key: v1-dependencies-{{ checksum "project.clj" }}
+          paths:
+            - ~/.m2
+      - save_cache:
+          key: v1-node-dependencies-{{ checksum "package.json" }}
+          paths:
+            - ~/node_modules
+
+  lint_code:
+    docker:
+      - image: circleci/clojure:lein-2.8.3
+    working_directory: ~/build
+    steps:
+      - restore_cache:
+          keys:
+            - v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+            - v1-dependencies-{{ checksum "project.clj" }}
+      - run: lein cljfmt check
+
+  verify_docs:
```

Comment:
That seems to be step building docs, but I could not see this step getting executed https://circleci.com/gh/Workiva/recide/93?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link 
so I guess my question is - is it something which we are going to use in future?